modules/data.atmosphere/R/download.NEONmet.R

```diff
@@ -0,0 +1,486 @@
+##' Download NEON Site Met CSV files
+##'
+##' @name download.NEONmet
+##' @title download.NEONmet
```

Comment:
The `@name` and `@title` tags are bad practice that we have, unfortunately, adapted somehow. I would strongly recommend removing them. The `@title` tag is redundant with what you already have as the function title ("Download NEON Site Met CSV files"), and the `@name` tag is read automatically from the function name. The only time you actually want to set the `@name` tag is when you specifically want the displayed function name to be different from the function being described, which is rarely the case.

The recommended structure for documentation (described [here](http://r-pkgs.had.co.nz/man.html#roxygen-comments)) is to not use tags for the introduction at all. Instead, the structure of the documentation should be:

```r
#' This is the function title
#'
#' This is a paragraph. This paragraph contains the function description.
#' It is technically the same as the @description tag, but the recommendation is to avoid
#' using that explicit tag.
#' 
#' This paragraph, and any paragraphs following, are the same as the @details tag. This is
#' a good place to put the NEON file documentation you have at the bottom.
#' 
#' This is another paragraph in the @details section. Etc, etc.
#'
#' @export
#' @param ...
```